wlc/__init__.py

```diff
@@ -18,13 +18,19 @@
 #
 """Weblate API client library."""
 
+import json
+import logging
 from copy import copy
 from urllib.parse import urlencode, urlparse
 
 import dateutil.parser
 import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
 
-__version__ = "1.10"
+log = logging.getLogger("wlc")
+
+__version__ = "1.11"
```

Comment:
1.10 was not yet released, so there is no need to increase version.

```suggestion
log = logging.getLogger("wlc")

__version__ = "1.10"
```